k8s.tf

```
@@ -104,7 +104,7 @@ resource "kubernetes_deployment_v1" "waggle_dance" {
           }
           liveness_probe {
             http_get {
-              path = "/actuator/health"
+              path = "/actuator/health/liveness"
```

```suggestion
              path = "/actuator/health"
```

removing this since it's required for another release